create_aio_app/template/{{cookiecutter.project_name}}/Makefile

```diff
@@ -19,10 +19,10 @@ bash:
 # Linters & tests
 
 mypy:
-	@docker-compose run --rm $(PROJECT_NAME)_app mypy $(PROJECT_NAME)
+	mypy $(PROJECT_NAME)
```

Comment:
i'm not sure that it is good idea. U need have `mypy` locally.

Reply:
You mostly will have it on your system and it is not problematic to install it. But after thinking more I guess you are right and it is easy to tune make command if you need so i will revert it.

Reply:
yes, thx